resources/x509_cert_gen/openssl-san.cnf

```diff
@@ -1,15 +1,13 @@
-[req]
+[ req ]
 x509_extensions = v3_req
 distinguished_name = req_distinguished_name
 
-[req_distinguished_name]
+[ req_distinguished_name ]
 
 [ v3_req ]
-
-# Extensions to add to a certificate request
-
 basicConstraints = CA:true
 keyUsage = nonRepudiation, digitalSignature, keyEncipherment
 subjectAltName = @alt_names
 
-[alt_names]
+[ alt_names ]
+DNS.1 = s10501-ip6.qe.couchbase.com
```

Comment:
you are not supposed to hard code the alt_names. Also I see spaces added in line #1  and #5. if spaces are not required, can you avoid checking this file ? If required, just remove the hard coded VM under alt_names and commit

Reply:
Also, can you run our regular regession jobs against ipv4 and make sure our current regression is broken.
Just run jobs against ipv4 for
SGW. functionaal tests: sanity is fine 
SGW upgrate test - test server 
SGW topology tests
Any one CBL functional tests: sanity is fine

One SGW functional job which you already ran against ipv6